desktop/core/src/desktop/js/ko/components/ko.shareGistModal.js

```diff
@@ -41,12 +43,52 @@ const TEMPLATE = `
         </button>
```

Comment:
The panel will look nicer if the copy button is on the right of the text input for the gist link.

Reply:
Something like this @JohanAhlen ? or down right (currently its down left for reference) and not inline?

<img width="593" alt="Screenshot 2021-04-28 at 1 02 01 PM" src="https://user-images.githubusercontent.com/42064744/116364559-ef78f180-a821-11eb-8191-30509de96ffe.png">

Reply:
Yes, I think that's better with the new form below. Thanks!

Reply:
Changes done!